argilla/src/argilla/_api/_http/_client.py

```diff
@@ -23,12 +23,8 @@ class HTTPClientConfig:
 
     api_url: str
     api_key: str
-    timeout: int = None
-
-    def __post_init__(self):
-        self.api_url = self.api_url
-        self.api_key = self.api_key
-        self.timeout = self.timeout or 60
+    timeout: int = 60
```

Comment:
@gabrielmbmb I think we can add this to the constants.py file.

Reply:
@gabrielmbmb The refactor of `timeout` and `retries` looks good to me. It's more consistent and avoids background passing with `**client_args`.

@davidberenstein1957 A few PRs have used constants in context lately. I think we should tackle this in a dedicated PR because using a single constant file is not universally good.